src/app/views/query-response/graph-toolkit/GraphToolkit.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { connect } from 'react-redux';
+
+import { Label } from 'office-ui-fabric-react';
+import { IQuery } from '../../../../types/query-runner';
+import { parseSampleUrl } from '../../../utils/sample-url-generation';
+import templates from './toolkit';
+
+// const templateMap: any = {
+//   '/me/planner/tasks' : 'https://mgt.dev/iframe.html?id=components-mgt-tasks--tasks',
+//   '/me': 'https://mgt.dev/iframe.html?id=components-mgt-person-card--person-card-hover',
+//   '/me/people' : 'https://mgt.dev/iframe.html?id=components-mgt-people--people',
+// };
+
+class GraphToolkit extends Component<any> {
+  constructor(props: any) {
+    super(props);
+  }
+
+  public lookupToolkitUrl = (sampleQuery: IQuery): any => {
+    if (sampleQuery) {
+      const { requestUrl, search } = parseSampleUrl(sampleQuery.sampleUrl);
+      const query = '/' + requestUrl + search;
+      const url: string = (templates as any)[query];
+      if (url) {
+        return url;
+      }
+    }
+    return null;
+  }
+
+  public render() {
+    const { sampleQuery } = this.props;
+    const toolkitUrl = this.lookupToolkitUrl(sampleQuery);
+
+    if (toolkitUrl) {
+      return (
+        <iframe width='100%' height='470px' src={toolkitUrl} />
```

Comment:
Is it possible to add a link bellow or above the iframe to open the sample in mgt.dev, similar to how the docs do it?

Something like: Open this example in mgt.dev

Reply:
Good suggestion

Reply:
I'm pointing the user to the exact example in mgt.dev. I hope you'll keep all examples with the same id as the ones in the story board.